.eslintrc.js

```diff
@@ -143,8 +143,9 @@ module.exports = {
 		'plugin:@wordpress/eslint-plugin/recommended',
 		'plugin:eslint-comments/recommended',
 		'plugin:storybook/recommended',
+		'plugin:react-hooks/recommended-legacy',
```

Comment:
Doesn't this have `react-compiler` turned off by default?

https://github.com/facebook/react/blob/main/packages/eslint-plugin-react-hooks/src/index.ts#L46